### PR TITLE
Fix dev start script

### PR DESCRIPTION
**Pull request by nessiLahav**
### What does this PR do?
Fix dev start script

#### Change log
- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

#### API Changes
- [ ] The [OpenAPI spec](https://github.com/cyberark/conjur-openapi-spec) has been updated to meet new API changes (or an issue has been opened), or
- [ ] The changes in this PR do not affect the Conjur API

### CODE REVIEW

File: dev/files/killConjurServer [hunk 1]
```diff
@@ -10,6 +10,10 @@ kill_recurse() {
 }
 
 conjur_server_pid=$(pgrep -f 'ruby /usr/local/bin/conjurctl server')
-if [ -n "$conjur_server_pid" ]; then
-  kill_recurse "$conjur_server_pid"
-fi
+#Iterate over conjur server process
```
Comment (tzheleznyak) [inline dev/files/killConjurServer:13]: Add space after #

**Review by tzheleznyak — APPROVED** [review]
left small comment. LGTM

File: dev/files/killConjurServer [hunk 2]
```diff
@@ -10,6 +10,10 @@ kill_recurse() {
...
+while IFS= read -r pid
```
Comment (Avi-Eyal) [inline dev/files/killConjurServer:14]: Why is it a `while` and not an `if?`
is it supposed to change on runtime? if so, is there a possibility for endless while loop?

**Review by Avi-Eyal — APPROVED** [review]
🚀

**Pull request merged**